lib/spack/spack/cmd/make_installer.py

```diff
@@ -31,33 +34,33 @@ def make_installer(parser, args):
     """
     if(sys.platform == 'win32'):
         output_dir = args.output_dir
-
+        cmake_spec = Spec('cmake')
+        cmake_spec.concretize()
+        cmake_path = os.path.join(cmake_spec.prefix, "bin", "cmake.exe")
+        cpack_path = os.path.join(cmake_spec.prefix, "bin", "cpack.exe")
         spack_source = args.spack_source
         if spack_source:
             if not os.path.exists(spack_source):
                 print("%s does not exist" % spack_source)
                 return
             else:
-                spack_source = os.path.abspath(spack_source)
-                spack_source = spack_source.replace("\\", "/")
+                spack_source = posixpath.abspath(spack_source)
 
         spack_version = args.spack_version
 
         here = os.path.dirname(os.path.abspath(__file__))
         source_dir = os.path.join(here, "installer")
+        posix_root = spack.paths.spack_root.replace('\\', '/')
+        spack_license = posixpath.join(posix_root, "LICENSE-APACHE")
 
-        spack_license = os.path.join(spack.paths.spack_root, "LICENSE-APACHE")
-        spack_license = spack_license.replace("\\", "/")
-
-        spack_logo = os.path.join(spack.paths.spack_root,
+        spack_logo = posixpath.join(posix_root,
                                   "share/spack/logo/favicon.ico")
-        spack_logo = spack_logo.replace("\\", "/")
 
         try:
             subprocess.check_call(
-                ('cmake -S "%s" -B "%s" -DSPACK_VERSION=%s '
+                ('"%s" -S "%s" -B "%s" -DSPACK_VERSION=%s '
                  '-DSPACK_SOURCE="%s" -DSPACK_LICENSE="%s" -DSPACK_LOGO="%s"')
-                % (source_dir, output_dir, spack_version, spack_source,
+                % (cmake_path, source_dir, output_dir, spack_version, spack_source,
```

Comment:
Is there a reason to use space-separated arguments to `check_call` rather than a list? The latter would prevent you from having to worry about escaping.